app/scripts/modules/core/src/managed/MarkArtifactAsBadModal.tsx

```diff
@@ -62,7 +62,12 @@ export interface IMarkArtifactAsBadModalProps extends IModalComponentProps {
 }
 
 export const showMarkArtifactAsBadModal = (props: IMarkArtifactAsBadModalProps) =>
-  showModal(MarkArtifactAsBadModal, props, { maxWidth: 750 });
+  showModal(MarkArtifactAsBadModal, props, { maxWidth: 750 }).then((result) => {
+    if (status === 'DISMISSED') {
```

Comment:
I can't quite figure out where `status` comes from.

Reply:
it comes from [the result object returned by `showModal()`](https://github.com/spinnaker/deck/blob/master/app/scripts/modules/core/src/presentation/modal/showModal.tsx#L54)

Reply:
Hmmm, I don't see `status` retrieved from `result` object here? I'm not sure why eslint didn't complain here.

Reply:
oopsie, yeah good catch this is a refactor gone bad. Really confused as to why this built or passed CI at all 😕

Reply:
[fixed](https://github.com/spinnaker/deck/pull/8738/commits/b7d42bb71460ab3983798d5ac0a7381316628ae0) — apparently there is such a thing as `window.status`, which is a string, and hence TS thought this was totally a-ok 🙄